src/views/tickets/view-link.php

```diff
@@ -40,18 +40,26 @@
 
 $counters = [];
 
-if ( 0 !== $rsvp_count ) {
-	$counters[] = sprintf( _n( '%d RSVP', '%d RSVPs', $rsvp_count, 'event-tickets' ), $rsvp_count );
+if ( 1 === $rsvp_count ) {
+	$counters[] = sprintf( _x( '%d %s', 'RSVP count singular', 'event-tickets' ), $rsvp_count, tribe_get_rsvp_label_singular( basename( __FILE__ ) ) );
 }
 
-if ( 0 !== $ticket_count ) {
-	$counters[] = sprintf( _n( '%d Ticket', '%d Tickets', $ticket_count, 'event-tickets' ), $ticket_count );
+if ( 1 < $rsvp_count ) {
```

Comment:
```suggestion
elseif ( 1 < $rsvp_count ) {
```

And can you move this block to after the end of the one above